docs/cvl/builtin.md

```diff
@@ -1,113 +1,195 @@
 Built-in Rules
 ==============
-The Certora Prover has built-in general-purpose rules targeted at finding known vulnerabilities.
-These rules can be verified on a contract out-of-the-box.
 
-Users can add built-in rules and other rule types in the same spec file.
+The Certora Prover has built-in general-purpose rules targeted at finding common
+vulnerabilities.  These rules can be verified on a contract without writing any
+contract-specific rules.
 
+Built-in rules can be included in any spec file by writing `use builtin rule
+<rule-name>;`.  This document describes the available built-in rules.
+
+```{todo}
+I would be in favor of removing the two "How ... is checked" sections; it seems
+like an implementation detail and I'm not sure that it's useful to users.  I
```

Comment:
Not all indeed, but I feel some users would appreciate a better understanding of the prover work